source/what-is-the-bi-connector.txt

```diff
@@ -38,22 +38,23 @@ business intelligence tools.
 System Components
 -----------------
 
-A complete BI system includes the following components:
+The |bi-short| deployment includes the following components:
 
 BI tool
-  Retrieve, visualize, and report on data from a database.
+  Visualization and reporting tool, such as Tableau and PowerBI.
```

Comment:
Do we want to link to them here? It is a PM question. I know that for [Partner Integrations docs in Atlas](https://docs.atlas.mongodb.com/partner-integrations/), we linked directly to any tools that are mentioned in that chapter. It might be the same case here but needs a PR oversight.

Reply:
I don't think so because these are arbitrary examples and we haven't set that standard, but I'll bring it up in tech review